http-api/src/lib.rs

```diff
@@ -486,6 +501,63 @@ async fn tree_handler(
     Ok(warp::reply::json(&response))
 }
 
+/// List all projects that delegate is a part of
+async fn delegates_projects_handler(ctx: Context, delegate: Urn) -> Result<impl Reply, Rejection> {
+    use radicle_daemon::git::identities::SomeIdentity;
+    #[derive(serde::Serialize)]
+    struct Project {
+        name: String,
+        description: String,
+        head: String,
+        urn: String,
+        delegates: Vec<PeerId>,
+    }
```

Comment:
Maybe pull this out and share it with `project_root_handler`?

Reply:
Yup, here https://github.com/radicle-dev/radicle-client-services/commit/796374d732981a55c1ed2b674c4fd69e84aafc47#diff-191ca8c6d151a1a4c5243319a8395c8a17c03c6e257bf4d796b004f19a0bf469R425-R433